New log file opened at Wed Aug 31 23:18:53 MDT 2016 by 42499@Chases-MacBook-Air.local
Taking next connection to: 64962
New client attempting to connect on port: 64963-64962
  Address = /10.0.1.3:64962
This client on port 64963 has uuid 70f14cac-329c-42d5-a7ee-61e5f13a6070 configName activator humanReadableName Activator protocolVersion 3 featureTags Vector()
Connected Clients: activator-70f14cac-329c-42d5-a7ee-61e5f13a6070
Taking next connection to: 64962
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 64962
New client attempting to connect on port: 49636-64962
  Address = /10.0.1.3:64962
Handshake exception on socket: 49636
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.net.SocketException: Connection reset
	at java.net.SocketInputStream.read(SocketInputStream.java:209)
	at java.net.SocketInputStream.read(SocketInputStream.java:141)
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246)
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265)
	at java.io.DataInputStream.readInt(DataInputStream.java:387)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 64962
New client attempting to connect on port: 49637-64962
  Address = /10.0.1.3:64962
This client on port 49637 has uuid de0cc346-6a51-431c-a979-04a2d51787a2 configName activator humanReadableName Activator protocolVersion 3 featureTags Vector()
Connected Clients: activator-de0cc346-6a51-431c-a979-04a2d51787a2
Taking next connection to: 64962
